frontend/jammy/handle_deb.go

```diff
@@ -285,7 +317,8 @@ func buildDepends(worker llb.State, sOpt dalec.SourceOpts, spec *dalec.Spec, tar
 		)
 
 		return in.Run(
-			installWithConstraints(debPath+"/*.deb", depsSpec.Name, customRepoOpts),
+			installWithConstraints(debPath+"/*.deb", depsSpec.Name, opts...),
+			customRepoOpts,
```

Comment:
Thank you for cleaning this up and addressing your review suggestion from before to add the `customRepoOpts` instead of threading through `install...`